### PR TITLE
Fix/generate from csv

### PR DESCRIPTION
**Pull request by Piranias**
Fix #Issue

**Changes proposed in this pull request**:
- Your_changes

The following steps were realized, as well (required):
- [x] Correct linting with `fake8 path_to_script`
- [ ] Check if tests pass locally (`pytest ./src/tests.py`)

Also the following steps were realized (if applies):
- [ ] Write docstrings to your code
- [ ] For new functionalities: Explain in readthedocs
- [ ] Write test(s) for your new patch of code

*Please mark above checkboxes as following:*
- [ ] Open
- [x] Done

### CODE REVIEW

File: generate_from_csv.py [hunk 1]
```diff
@@ -64,6 +64,10 @@ def generate_from_csv(args):
             export_filename = None
         input = assign_vehicle_id(input, args.min_standing_time, export_filename)
 
+    if "connect_cs" not in input[0].keys():
+        warnings.warn("Column 'connect_cs' is not available, all vehicles will be connected.")
```
Comment (jakobgemassmer) [inline generate_from_csv.py:68]:
```suggestion
        warnings.warn("Column 'connect_cs' is not available. Vehicles will be connected to a charging station after every trip.")
```

File: tests/test_data/input_test_generate/generate_from_csv_template4.csv [hunk 2]
```diff
@@ -1,7 +1,7 @@
-departure_time,arrival_time,vehicle_type,distance
-2021-09-16 04:22:18,2021-09-16 19:45:41,AB-OPP,5
-2021-09-17 02:59:49,2021-09-17 22:43:10,AB-OPP,8
-2021-09-18 04:51:34,2021-09-18 20:52:25,AB-OPP,10
-2021-09-16 04:23:23,2021-09-16 20:05:36,AB-OPP,4
-2021-09-17 03:37:27,2021-09-17 20:45:32,AB-OPP,12
-2021-09-18 03:50:13,2021-09-18 22:38:46,AB-OPP,1
+departure_time,arrival_time,vehicle_type,distance,connect_cs
```
Comment (jakobgemassmer) [inline tests/test_data/input_test_generate/generate_from_csv_template4.csv:1]: Isn't the vehicle_id missing here?

**Review by jakobgemassmer — COMMENTED** [review]
Good improvement! Only minor comments.

File: tests/test_data/input_test_generate/generate_from_csv_template4.csv (hunk 2 above)
Comment (Piranias) [reply tests/test_data/input_test_generate/generate_from_csv_template4.csv:1]: the vehicle_id is optional. If there is no column vehicle_id it is assigned automatically via the principle "first in, fist out". A minimum standing time can be defined in the input arguments

**Pull request merged**